packages/venia-styleguide/src/components/Article/Article.css

```diff
@@ -0,0 +1,4 @@
+.title {
+    font-size: var(--venia-global-text-fontSize-1100);
```

Comment:
This feels like it should use a typography token for heading but I didn't find one that used `1100`.